test/db/analysis/tricore

```diff
@@ -686,10 +706,9 @@ EXPECT=<<EOF
 .---------------------------------------------------------.   .---------------.
 |  0x800004f6                                             |   |  0x800004f2   |
 | mov d4, #0x14                                           |   | mov d15, #0   |
-| ; dbg.malloc                                            |   | j #0x8000054c |
-| ; void *malloc(size_t size)                             |   `---------------'
-| call #0x80000afc                                        |       v
-| mov.aa a15, a2                                          |       |
+| ; void *malloc(size_t size)                             |   | j #0x8000054c |
```

Comment:
What about also not showing `#` in case of jumps to the address?

Reply:
It's a bit of a hassle.

If we simply remove all "#" from the opstr of all `JMP`, `CALL`, and `UJMP` instructions, it seems to affect other instructions.

I was trying to work at the Operands level, but realized that our replacement for disassemble also works at the string level.

Reply:
Operating on the tokens level should help solve this too, see @Rot127's comment.